src/extensions/attributes/index.js

```diff
@@ -30,36 +29,23 @@ function addAttributes( settings ) {
  * @param {function|Component} BlockEdit Original component.
  * @return {string} Wrapped component.
  */
-const withAttributes = createHigherOrderComponent( ( BlockEdit ) => {
-	return ( props ) => {
-		const {
-			attributes,
-			setAttributes,
-		} = props;
+const withAttributes = createHigherOrderComponent(
+	BlockEdit => props => {
+		const { name: blockName } = props;
 
-		if ( typeof attributes.coblocks === 'undefined' ) {
-			attributes.coblocks = [];
-		}
+		if ( allowedBlocks.includes( blockName ) ) {
+			props.attributes.coblocks = props.attributes.coblocks || {};
```

Comment:
We don't want every block to have a `coblocks` attribute so we only set it up if the current block is an allowed block.